src/equations/compressible_navier_stokes_3d.jl

```diff
@@ -78,7 +78,7 @@ w_2 = \frac{\rho v_1}{p},\, w_3 = \frac{\rho v_2}{p},\, w_4 = \frac{\rho v_3}{p}
 """
 struct CompressibleNavierStokesDiffusion3D{GradientVariables, RealT <: Real,
                                            E <: AbstractCompressibleEulerEquations{3}} <:
-       AbstractCompressibleNavierStokesDiffusion{3, 5}
+       AbstractCompressibleNavierStokesDiffusion{3, 5, GradientVariables}
```

Comment:
So if I understand correctly, this modification sets `GradientVariablesConservative` to be the default within `equations_parabolic.jl` only for the Laplace diffusion whereas for Navier-Stokes we still need to pass whichever `gradient_variables` we want into the constructor?

For completeness, would it be necessary to add an appropriate `convert_derivative_to_primitive` function for the `GradientVariablesConservative` in the Navier-Stokes file?

Reply:
No, for Navier-Stokes the constructor defaults to `GradientVariablesPrimitive`.

I don't think we plan on adding conservative gradient variables, so I'd lean towards not implementing `convert_derivative_to_primitive` for conservative gradient variables.